obvious-ci.docker/linux64_obvci/Dockerfile

```diff
@@ -23,6 +23,20 @@ RUN yum update -y && \
                    mesa-libGL-devel && \
     yum clean all
 
+# Install devtoolset 2.
+RUN yum update -y && \
+    yum install -y \
+                   centos-release-scl \
+                   yum-utils && \
+    yum-config-manager --add-repo http://people.centos.org/tru/devtools-2/devtools-2.repo && \
+    yum update -y && \
+    yum install -y \
+                   devtoolset-2-binutils \
+                   devtoolset-2-gcc \
+                   devtoolset-2-gcc-c++ \
+                   devtoolset-2-gcc-gfortran && \
```

Comment:
Going to rip this out. We may need to add it back in the future if we use `gfortran` from devtoolsets too.